### ISSUE TITLE
Memory unit and monitor on system level

### ISSUE THREAD

**Issue by iwangtoknow** [jupyter-server/jupyter-resource-usage#229]
Hi there,

I have a question and a suggestion.

Question: how can I change the memory unit shown in the status bar? Now I have 1028096.00 MB I would like to see it in GB.

Suggestion: I am working on a node shared by multiple users. I would like to have the monitor also on the system instead of just my usage. Like the status shown in htop.

Cheers

**Comment by perllaghu** [comment]
Interesting... I get the same
I hadn't noticed until now!

**Comment by iwangtoknow** [comment]
@perllaghu Hi, great you see it. Do you have any updates?

**Comment by perllaghu** [comment]
No..... I was going to look at this once I'd got a quieter period at work

**Comment by iwangtoknow** [comment]
Hi,
I don't know what happened but. The memory unit is now in GB.
Cheers

**Comment by mahendrapaipuri** [comment]
@iwangtoknow The format is automatically chosen based on [memory usage](https://github.com/jupyter-server/jupyter-resource-usage/blob/7175826f0c2e5cff40bfb050d6c3465d002391af/packages/labextension/src/format.ts#L8-L23).

**Comment by iwangtoknow** [comment]
@mahendrapaipuri Thanks for the clarification. It was in MB for a few days, afterwards, it was changed to GB. Do you have any comments/solutions on the system-level resource monitor?

Cheers

**Comment by iwangtoknow** [comment]
@mahendrapaipuri the function of unit auto-selection is not stable, today it jumps back to MB.

**Comment by nimpo** [comment]
+1 for monitoring memory availability for the whole system. In a multi-user environment, having sight of the system's available memory is useful especially is some of the other notebooks are being greedy.
